SSA/Experimental/ErasedContext.lean

```diff
@@ -39,10 +39,10 @@ def ofList : List Ty → Ctxt Ty :=
   -- Erased.mk
   fun Γ => Γ 
 
+-- Why was this noncomutable? (removed it to make transformation computable)
 @[simp]
```

Comment:
Because @ChrisHughes24 suggested to use a non-computable List such that the data-structure can be eliminated when executing the code.

Reply:
At some point we seem to have got rid of any use of `Erased` in the `Erased` context file. So we're not actually using erased contexts any more

Reply:
@alexkeizer is working on getting it back.

Reply:
https://github.com/opencompl/ssa/tree/ctxt-enhancements

Reply:
What you probably want to do, is to have the various `mk*` types that need to compute with the context take a `List Ty` instead of the erased `Ctxt Ty`